src/content/docs/data-apis/ingest-apis/event-api/incident-event-rest-api.mdx

```diff
@@ -0,0 +1,310 @@
+
+### Example NRDB query for incident events [#nrdb-example]
+
+You can query the New Relic database with [Nerdgraph, our GraphQL API](/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph/).
+
+```json
+{
+  actor {
+    account(id: <var>YOUR_ACCOUNT_ID</var>) {
+      nrql(query: "SELECT * FROM NrAiInternalIncident since 3 days ago") {
+        results
+      }
+    }
+  }
+}
+
+"results": [
+  {
+    "timestamp": 1641910123824,
+    "totalViolations": 3,
+    "violationIds": "[416e2e55069764086ad83e97a1160be0cb4c545b7950522f7c86baaa2a11b9b6]",
+    "triggeredAt": 1641909163980,
+    "entitiesData.ids": "testEntity123",
+    "entitiesData.types": "unknown",
+    "labelsHash": "91f938318e899dcd62965e2724548f4844f2898f3cf1c5411397ecc0eec87fc1",
+    "annotations.description": "[\"The latency is above threshold of 500000 MS\"]",
+    "labels.originalAccountIds": "<var>YOUR_ACCOUNT_ID</var>",
+    "entitiesData.names": "testEntity",
+    "entitiesData.entities": "{\"id\":\"testEntity123\",\"name\":\"testEntity\",\"type\":\"unknown\"}",
+    "priority": "HIGH",
+    "labels.serviceId": "5",
+    "labels.environment": "testing",
+    "closedAt": "",
+    "updatedAt": 1641910123824,
+    "annotations.title": "[\"Test\",\"Test2222\",\"Test3\"]",
+    "nrAccountId": <var>YOUR_ACCOUNT_ID</var>,
+    "accumulations": "{\"source\":[\"rest\"],\"origin\":[\"luna\"],\"entity_guid\":[\"testEntity123\"],\"tag.stackTrace\":[\"some stack trace...\"]}",
+    "labels.accountIds": "<var>YOUR_ACCOUNT_ID</var>",
+    "createdAt": 1641909163997,
+    "priority.order": 2,
+    "labels.aggregationKeys": "416e2e55069764086ad83e97a1160be0cb4c545b7950522f7c86baaa2a11b9b6",
+    "accumulations.origins": "[\"luna\"]",
+    "accumulations.entity_guid": "[\"testEntity123\"]",
+    "incidentId": "a06ffb92-2f7e-473b-953f-151ff4777cb0",
+    "labels.errorId": "10543",
+    "dataMLModules": "{}",
+    "triggerEvent": "VIOLATION_ADDED",
+    "accumulations.tag.stackTrace": "[\"some stack trace...\"]",
+    "isIint": false,
+    "accumulations.sources": "[\"rest\"]",
+    "entitiesData": "{\"name\":\"testEntity\",\"id\":\"testEntity123\",\"type\":\"unknown\",\"entities\":\"{\\\"id\\\":\\\"testEntity123\\\",\\\"name\\\":\\\"testEntity\\\",\\\"type\\\":\\\"unknown\\\"}\"}",
+    "annotations": "{\"title\":[\"Test\",\"Test2222\",\"Test3\"],\"description\":[\"The latency is above threshold of 500000 MS\"]}",
+    "labels": "{\"accountId\":\"XXXXXXX\",\"originalAccountId\":\"XXXXXXX\",\"entityId\":\"testEntity123\",\"entityName\":\"testEntity\",\"entityType\":\"unknown\",\"aggregationKey\":\"416e2e55069764086ad83e97a1160be0cb4c545b7950522f7c86baaa2a11b9b6\",\"serviceId\":\"5\",\"errorId\":\"10543\",\"environment\":\"testing\"}",
+    "state": "CREATED"
+  }
+]
+```
+
+## API specifications [#api-specs]
+
+For descriptions of our value types, see [our event API JSON guidelines](/docs/data-apis/ingest-apis/introduction-event-api/#json-guidelines).
+
```

Comment:
It feels like we need a little more preface to these things. What is the relationship between these and the Event API JSON guidelines just mentioned?